modules/tinymce/src/core/main/ts/api/Options.ts

```diff
@@ -859,6 +859,11 @@ const register = (editor: Editor): void => {
     default: true
   });
 
+  registerOption('fullscreen_trap_focus', {
+    processor: 'boolean',
+    default: false
```

Comment:
Maybe we should consider this be true for v8 seems like something you normally don't want to happen.

Reply:
I asked for this to be an undocumented option that the fullscreen plugin can set in order to avoid core needing to know the plugin exists and what it's capable of. The alternative would be to pull fullscreen into the core, which while more convenient for fixes like this probably isn't the best idea.

A better name would probably be `trap_focus`, then it's a capability that can be enabled and disabled as appropriate.

Reply:
Fixed that, now it has been renamed to `trap_focus`.